test/FileService.Mappers.UnitTests/FileService.Mappers.UnitTests.csproj

```diff
@@ -1,5 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <NoWin32Manifest>true</NoWin32Manifest>
+  </PropertyGroup>
+
```

Comment:
What is this for?

Reply:
I guess it was added automatically with resources